api/mongo-migrations/1671458857144-key-state-personnel-mmis.js

```diff
@@ -0,0 +1,63 @@
+const logger = require('../logger')(
+  'mongoose-migrate/migrate-key-state-personnel-mmis'
+);
+const { setup, teardown } = require('../db/mongodb');
+const { APD } = require('../models');
+
+/**
+ * Update HITECH APDs to have a default 90-10 split for key state personnel
+ */
+async function up() {
+  // Grab all APDs
+  await setup();
+  const apds = await APD.find({}).lean();
+
+  // Create new object with updated keyPersonnel.split
+  const updatedApds = apds
+    .filter(apd => apd.__t === 'HITECH')
+    .map(apd => {
```

Comment:
you can import HITECH instead of APD and then it will only retrieve the HITECH apds and you won't have to filter